examples/nlp/dialogue_state_tracking/dialogue_state_tracking_sgd.py

```diff
@@ -209,8 +209,8 @@
     "--state_tracker",
     type=str,
     default='baseline',
-    choices=['baseline', 'ret_sys_act'],
-    help="Specifies the state tracker mode",
+    choices=['baseline', 'nemotracker'],
```

Comment:
please update the sgd docs to explain what this new tracker includes and how it's different from the baseline one.

Reply:
Done